### PR TITLE
Remove broken links from SSA notebooks

### PR DESCRIPTION
**Pull request by jkwashbourne**
Links dont work on static pages, I have removed. No other changes.

### CODE REVIEW

File: examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb [hunk 1]
```diff
@@ -131,6 +129,37 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: matplotlib in /cpfs/lfs03/ESDRD/wask/local/lib/python3.8/site-packages (3.2.1)\n",
```
Comment (FabioLuporini) [inline examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb:148]: these are system-specific paths specific of your machine, so we probably don't want them here

Comment (jkwashbourne) [reply examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb:148]: pretty sure that is output? I have #NBVAL_INGNORE_OUTPUT ignore at top of the cell.

Comment (FabioLuporini) [reply examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb:148]: yeah but do we really want your paths (e.g. `../.. /wask/...`) `/cpfs/lfs03/ESDRD/wask/local/lib/python3.8/site-packages` in the cell output?

Comment (jkwashbourne) [reply examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb]: OK, added ```%%capture``` at the top of the cell.

**Pull request merged**